addon/components/discover-page/template.hbs

```diff
@@ -31,8 +31,9 @@
             <div class="col-xs-12 col-sm-8 col-sm-offset-2">
                 {{!SEARCH BAR}}
                 <div class="input-group input-group-lg">
-                    <span class="searchBoxOverride" aria-label={{t 'eosf.components.discoverPage.searchInputBox'}}>
+                    <span class="searchBoxOverride">
                         {{input
+                            (html-attributes aria-label='Search')
```

Comment:
This has to be a bare string rather than a translated one due to limitations on `ember-component-attributes`. Though I suspect this will be merged into ember-osf-web before we get to translating preprints to a new language.

Reply:
https://github.com/mmun/ember-component-attributes#limitations